build/ci-build.yml

```diff
@@ -0,0 +1,78 @@
+name: Build Arcus.WebApi CI
+
+trigger:
+  - master
+pr:
+  - master
+
+variables:
+  - group: 'Arcus - GitHub Package Registry'
+  - group: 'Build Configuration'
+
+  # Always use fixed version for .NET Core SDK
+  - name: 'DotNet.Sdk.Version'
+    value: '2.2.105'
+  - name: 'Project'
+    value: 'Arcus.WebApi'
+
+# Building the project:
+# - Download correct .NET Core SDK
+# - .NET Core build
+# - Publish build artifacts
+stages:
+  - stage: Build
+    jobs:
+      - job: Compile
+        pool:
+          vmImage: 'ubuntu-16.04'
+        steps:
+          - task: DotNetCoreInstaller@0
+            displayName: 'Import .NET Core SDK ($(DotNet.Sdk.Version))'
+            inputs:
+              version: '$(DotNet.Sdk.Version)'
+          - task: DotNetCoreCLI@2
+            displayName: 'Compile'
+            inputs:
+              projects: 'src/*.sln'
+              arguments: '--configuration $(Build.Configuration)'
+          - task: CopyFiles@2
+            displayName: 'Copy build artifacts'
+            inputs:
+              contents: '**/?(bin|obj)/**'
+              targetFolder: '$(Pipeline.Workspace)/build'
+          - task: PublishPipelineArtifact@0
+            displayName: 'Publish build artifacts'
+            inputs:
+              targetPath: '$(Pipeline.Workspace)/build'
+              artifactName: Build
+
+  # Testing unit tests of the project:
```

Comment:
Not sure if we need these comments though as the YAML itself serves as living documentation of our build.

What do you think @fgheysels 	?

Reply:
For me, they can be removed; should the tasks in the pipeline change, one must also make sure that the documentation is kept in-sync, and I think that this is easily forgotton :)

Reply:
That's my concern as well!